packages/custom_lint/lib/custom_lint.dart

```diff
@@ -89,24 +88,24 @@ void _renderLints(
   List<AnalysisErrorsParams> lints, {
   required Directory workingDirectory,
 }) {
-  final errors = lints
-      .expand(
-        (lintsForFile) => lintsForFile.errors
-          ..sort((a, b) {
-            final lineCompare =
-                a.location.startLine.compareTo(b.location.startLine);
-            if (lineCompare != 0) return lineCompare;
-            final columnCompare =
-                a.location.startColumn.compareTo(b.location.startColumn);
-            if (columnCompare != 0) return columnCompare;
-
-            final codeCompare = a.code.compareTo(b.code);
-            if (codeCompare != 0) return codeCompare;
-
-            return a.message.compareTo(b.message);
-          }),
-      )
-      .sortedBy((e) => _relativeFilePath(e.location.file, workingDirectory));
+  final errors = lints.expand((lintsForFile) => lintsForFile.errors).toList()
```

Comment:
Let's keep using .sorted if we're going to use `toList()`.

But we can do it in two lines if that's more readable.

Reply:
Done